src/components/user-profile.component.js

```diff
@@ -86,8 +86,8 @@ export const UserProfile = ({
       <ImageZoom
         uri={{
           uri: initialUser.avatar_url
-            ? initialUser.avatar_url
-            : user.avatar_url,
+            ? `${initialUser.avatar_url}&cache=${initialUser.updated_at}`
```

Comment:
I think `lastModified` might be the best query name to use so it's as close as possible to the `last-modified` header we would pass to the API.